src/main/java/seedu/linkedout/model/applicant/Applicant.java

```diff
@@ -20,18 +20,20 @@
     private final Email email;
 
     // Data fields
-    private final Address address;
+    private final Stage stage;
+    private final Job job;
     private final Set<Skill> skills = new HashSet<>();
 
     /**
      * Every field must be present and not null.
      */
-    public Applicant(Name name, Phone phone, Email email, Address address, Set<Skill> skills) {
-        requireAllNonNull(name, phone, email, address, skills);
+    public Applicant(Name name, Phone phone, Email email, Job job, Stage stage, Set<Skill> skills) {
+        requireAllNonNull(name, phone, email, job, skills);
```

Comment:
Do we need to add stage to this check as well?
```suggestion
        requireAllNonNull(name, phone, email, job, stage, skills);
```

Reply:
a972860c86657043b96a8bc0fbd3c3823566ffc4